docs/src/julia_syntax.md

```diff
@@ -75,8 +75,27 @@ JuliaBUGS provides a Julian interface inspired by Turing.jl's model macro syntax
 
 ### The `@model` Macro
 
+The `@model` macro provides a Julia-native interface for defining probabilistic models using external type definitions.
+
+#### External Type Definition with `@of`
```

Comment:
I'm not very keen on the name `of`. It's creating X of Y, but it's not clear from context or the name what X and Y can be. For instance, in some languages, `of` is used in names of functions that convert from one type to another, as in `String.of_int` (from OCaml) that turns an int to a string, but that's not what's happening here.

Would something like `parameter_type` work? I know its more verbose, but it would make it much more concrete and obvious what the function/macro is doing.